packages/server/src/GraphQLCache.js

```diff
@@ -411,23 +414,50 @@ export class GraphQLCache {
   ): Promise<?GraphQLSchema> => {
     const projectConfig = this._graphQLConfig.getProjectConfig(appName);
 
-    if (!projectConfig || !projectConfig.schemaPath) {
+    if (!projectConfig) {
       return null;
     }
 
     const projectName = appName || 'undefinedName';
-
     const schemaPath = projectConfig.schemaPath;
-    const schemaCacheKey = `${schemaPath}:${projectName}`;
+    const endpointInfo = this._getDefaultEndpoint(projectConfig);
```

Comment:
nit:
```js
const {endpointName, endpoint} = this._getDefaultEndpoint(projectConfig);
```

Reply:
Oh, it looks like `_getDefaultEndpoint` can return something that is falsy so destructuring here could possibly throw an error which is why I think it's handled below.